service/history/workflow/metrics.go

```diff
@@ -46,68 +46,41 @@ func emitWorkflowHistoryStats(
 	countScope.RecordDistribution(metrics.HistoryCount, historyCount)
 }
 
-func emitWorkflowExecutionStats(
+func emitMutableStateStatus(
 	metricsClient metrics.Client,
+	scope int,
 	namespace string,
-	stats *persistence.MutableStateStats,
-	executionInfoHistorySize int64,
+	stats *persistence.MutableStateStatus,
 ) {
-
 	if stats == nil {
 		return
 	}
 
-	sizeScope := metricsClient.Scope(metrics.ExecutionSizeStatsScope, metrics.NamespaceTag(namespace))
-	countScope := metricsClient.Scope(metrics.ExecutionCountStatsScope, metrics.NamespaceTag(namespace))
+	sizeScope := metricsClient.Scope(scope, metrics.NamespaceTag(namespace))
+	countScope := metricsClient.Scope(scope, metrics.NamespaceTag(namespace))
```

Comment:
why 2 identical scopes? can it be one variable?

Reply:
that was the original logic, but good catch

Reply:
actually, original logic was to diff scope, my fault